### PR TITLE
Sprint2

### PR DESCRIPTION
**Pull request by Kv-Khushi**

### CODE REVIEW

File: Restaurants/src/main/java/com/restaurants/entities/Restaurant.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    private String restaurantAddress;
+
+    private Long contactNumber;
+
+    private String restaurantDescription;
+
+    private Long openingHour;
+
+    private String restaurantImages;
```
Comment (Yogesh-Kanare-Nucleusteq) [inline Restaurants/src/main/java/com/restaurants/entities/Restaurant.java]: check the datatype and use appropriate datatype

File: Restaurants/src/main/java/com/restaurants/entities/RestaurantMenu.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    private String description;
+
+    private Boolean vegNonVeg;
+
+    private Long categoryId;
+
+    private Long restaurantId;
+
+    private String imageUrl;
```
Comment (Yogesh-Kanare-Nucleusteq) [inline Restaurants/src/main/java/com/restaurants/entities/RestaurantMenu.java]: check the datatype and use appropriate datatype

File: Restaurants/src/main/java/com/restaurants/indto/RestaurantRequest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    private String restaurantAddress;
+
+    @NotNull
+    private Long contactNumber;
+
+    private String restaurantDescription;
+
+    private Long openingHour;
+
+    private String restaurantImages;
```
Comment (Yogesh-Kanare-Nucleusteq) [inline Restaurants/src/main/java/com/restaurants/indto/RestaurantRequest.java]: check the datatype and use appropriate datatype

File: Restaurants/src/main/java/com/restaurants/outdto/RestaurantResponse.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+    private String restaurantAddress;
+
+    private Long contactNumber;
+
+    private String restaurantDescription;
+
+    private Long openingHour;
+
+    private String restaurantImages;
```
Comment (Yogesh-Kanare-Nucleusteq) [inline Restaurants/src/main/java/com/restaurants/outdto/RestaurantResponse.java]: check the datatype and use appropriate datatype

File: Restaurants/src/main/java/com/restaurants/outdto/UserDto.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+    private Long phoneNumber;
+
+    private String userName;
+
+    private String userEmail;
+
+    private String userPassword;
+
+    private Long wallet;
```
Comment (Yogesh-Kanare-Nucleusteq) [inline Restaurants/src/main/java/com/restaurants/outdto/UserDto.java]: use proper naming convention

**Review by Yogesh-Kanare-Nucleusteq — COMMENTED** [review]
solve the given review

File: Restaurants/src/main/java/com/restaurants/entities/RestaurantMenu.java (hunk 2 above)
Comment (Kv-Khushi) [reply Restaurants/src/main/java/com/restaurants/entities/RestaurantMenu.java]: Okay

File: Restaurants/src/main/java/com/restaurants/controller/FoodCategoryController.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+@RestController
+@RequestMapping("/foodCategories")
+public class FoodCategoryController {
+
+    @Autowired
+    private FoodCategoryService foodCategoryService;
+
+
+    @PostMapping("/add")
```
Comment (Yogesh-Kanare-Nucleusteq) [inline Restaurants/src/main/java/com/restaurants/controller/FoodCategoryController.java:38]: use proper naming convention for post url

**Pull request merged**